test/integration/suite_test.go

```diff
@@ -102,8 +122,10 @@ var _ = BeforeSuite(func() {
 	Expect(err).ToNot(HaveOccurred())
 
 	dnsProvider := &dns.FakeProvider{}
+	certificates := tls.NewService(k8sManager.GetClient(), "glbc-ca")
 	dns := dns.NewService(k8sManager.GetClient())
-	plc := NewTestOCMPlacer()
+	plc := placement.NewOCMPlacer(k8sManager.GetClient())
+	testPlc := NewTestOCMPlacer()
```

Comment:
Let's capture a follow up issue to avoid 2 different placer impls if possible.
There's some important logic in the real placer that the gateway controller relies on before we even expect ocm to be running e.g. the ManifestWork parsing of statusfeedback rules.

Reply:
two fake placers will be addressed here https://github.com/Kuadrant/multicluster-gateway-controller/issues/282 - they will be joined into one big happy fake placer - no need for the follow-up task if I understand things correctly.

Reply:
I have created issue #330

Reply:
I'd have expected this to be done as part of this move, but a follow up is fine